client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss

```diff
@@ -491,6 +491,15 @@ $design-button-primary-color: rgb( 17, 122, 201 );
 		}
 	}
 
+	.generated-design-picker__previews-scaled {
+		position: absolute;
+		top: 0;
+		right: 0;
+		width: 1160px;
```

Comment:
I think our goal is to align the width with the mShots, so the width have to the same as the vpw of the mShot (1600px)